nemo/core/neural_types/neural_type.py

```diff
@@ -73,6 +78,19 @@ def __init__(self, axes: Optional[Tuple] = None, elements_type: ElementType = Vo
             self.axes = None
         self.optional = optional
 
+    def __repr__(self) -> str:
```

Comment:
NeuralType already has a str(). Should we be printing this or str()?

Reply:
good catch - removed that, since for now str() and repr() should be the same